### DISCUSSION TITLE
Hide banners in focus mode

### DISCUSSION THREAD

**Discussion by lukew3** [monkeytypegame/monkeytype#2751] [Ideas] [answered]
Typing with banners visible is uncomfortable, and I don't want to have to click close on the banners every time there is an update. I suggest that either banners are removed when entering focus mode or are just hidden in focus mode and restored after.
![Screenshot from 2022-03-22 10-56-17](https://user-images.githubusercontent.com/47042841/159511483-2cdea517-5255-483b-bc45-926deece6b22.png)

**Comment by Miodec** [comment, marked as answer]
Good idea. Added issue.
